### ISSUE TITLE
Error 600 in readncfile

### ISSUE THREAD

**Issue by jdrenken** [NetCDF4Excel/project#18]
Hello,

While attempting to look at a recent .nc file I received the above error.
![ncerror](https://user-images.githubusercontent.com/12485005/27357038-5169b386-55d6-11e7-8dbf-44be02b2bc26.PNG)

**Comment by swordsgnat** [comment]
Yeah, I have the same issue, with files from [this](https://data.nodc.noaa.gov/ncei/wod/1800/) database, though other databases seem to work properly.

**Comment by Abdelaalmohamed1980** [comment]
I have the same issue

**Comment by JWO123** [comment]
Same here
